src/parameters/update_cost_parameters.jl

```diff
@@ -71,15 +71,15 @@ _get_parameter_field(
     args...;
     kwargs...,
 ) =
-    PSY.get_decremental_offer_curves(args...; kwargs...)
+    get_input_offer_curves(args...; kwargs...)
 
 _maybe_tuple(::StartupCostParameter, value) = Tuple(value)
 _maybe_tuple(::ShutdownCostParameter, value) = value
 _maybe_tuple(::AbstractCostAtMinParameter, value) = value
 
 function handle_variable_cost_parameter(
     param::Union{StartupCostParameter, ShutdownCostParameter, AbstractCostAtMinParameter},
-    op_cost::PSY.MarketBidCost,
+    op_cost::OfferCurveCost,
```

Comment:
I'm guessing OfferCurveCost is the supertype of MarketBid and ImportExport right?